.github/workflows/release.yml

```
@@ -0,0 +1,16 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ stdlib-release-pipeline ]
+
+jobs:
+  call_workflow:
+    name: Run Release Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/release-package-connector-template.yml@main
+    secrets: inherit
+    with:
+      package-name: docusign
```

Check similar places.

```suggestion
      package-name: docusign.dsesign
```